Slim/Web/Pages/Playlist.pm

```diff
@@ -177,6 +177,7 @@ sub playlist {
 	$titleFormat =~ /\bWORK\b/     && ($tags .= 'b');
 	$titleFormat =~ /\bGROUPING\b/     && ($tags .= 'h');
 	$titleFormat =~ /\bSUBTITLE\b/     && ($tags .= 'z');
+	$titleFormat =~ /\bPERFORMANCE\b/     && ($tags .= '1');
```

Comment:
`DISCSUBTITLE`?

Reply:
DISCSUBTITLE is driven by the same tag as DISC, so I don't think this is necessary.

Reply:
Which means that you can't request disc subtitles specifically without requesting the disc count? And you'd get the subtitle even if you didn't request it, but you requested discs instead? 

I was hesitant to find this good (well, still am...). But considering the lack of tags we run into, and the minor side effect this should cause, we should be good.